paasta_tools/cli/cmds/validate.py

```diff
@@ -149,6 +165,31 @@ def get_service_path(service, soa_dir):
             return None
 
 
+def path_to_soa_dir_service(service_path):
+    soa_dir = os.path.dirname(service_path)
+    service = os.path.basename(service_path)
+
+    return soa_dir, service
+
+
+def validate_chronos(service_path):
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    instance_type = 'chronos'
+
+    for cluster in list_clusters(service, soa_dir, instance_type):
+        for instance in list_all_instances_for_service(service, instance_type, soa_dir):
+            cjc = load_chronos_job_config(service, instance, cluster, False, soa_dir)
+            (checks_passed, check_msgs) = cjc.validate()
+
+            # Remove duplicate check_msgs
+            unique_check_msgs = list(OrderedDict.fromkeys(check_msgs))
```

Comment:
- why do you need this? What duplicated are there going to be? check_msgs is an array of strings afaict, so those strings are always going to be unique?
- we don't care about ordering, so list(set(check_msgs)) would be fine here.